### ISSUE TITLE
form.pauseValidation() prevents form subscriber from notifying

### ISSUE THREAD

**Issue by LukeAVanDrie** [final-form/final-form#383]
### Are you submitting a **bug report** or a **feature request**?

I'm not sure if this behavior is considered a bug or not. I don't understand why these features are so highly coupled. Thank you in advance for any help finding a workaround or letting me know the justification for this design choice.

### What is the current behavior?

When validation is paused, the form subscriber no longer notifies me of form updates. I don't understand why notification is locked behind the `validationPaused` flag.

```
let scheduleNotification: boolean = false
const notifyFormListeners = () => {
  if (notifying) {
    scheduleNotification = true
  } else {
    notifying = true
    callDebug()
    if (!inBatch && !validationPaused) {
      const { lastFormState } = state
      const nextFormState = calculateNextFormState()
      if (nextFormState !== lastFormState) {
        state.lastFormState = nextFormState
        notify(
          state.subscribers,
          nextFormState,
          lastFormState,
          filterFormState
        )
      }
    }
    notifying = false
    if (scheduleNotification) {
      scheduleNotification = false
      notifyFormListeners()
    }
  }
}
```

### What is the expected behavior?

I would expect the form subscriber to still notify me of form updates even while validation is paused.

### What's your environment?

final form: v4.20.1

### Other information

I'm bumping this from my [StackOverflow post](https://stackoverflow.com/questions/64121128/why-does-form-pausevalidation-prevent-the-form-subscriber-from-notifying). There are code references in that post.

**Comment by Renddslow** [comment]
It appears that this is related to a change made with #118 and has been previously reported with no additional comments in #129

**Comment by erikras** [comment]
Fix released in [`v4.20.2`](https://github.com/final-form/final-form/releases/tag/v4.20.2).
